templates/vue/api/fooFetch.js

```diff
@@ -0,0 +1,31 @@
+import SubmissionError from '../error/SubmissionError';
+import { API_HOST, API_PATH } from './_entrypoint';
+
+const jsonLdMimeType = 'application/ld+json';
+
+export default function {{{ lc }}}Fetch(url, options = {}) {
```

Comment:
why do we need a `fetch` utility per resource? Can't this be common to every resource?

Reply:
Yes of course !